jobbergate-docs/src/implementation_details.rst

```diff
@@ -0,0 +1,27 @@
+=======================
+Implementations Details
+=======================
+
+Jobbergate-API
+==============
+
+Storage
+-------
+
+Besides the information keept in the database, application and job-scripts files are stored on a S3 Bucket as tarfiles and text files, respectively.
+The bucket name is ``jobbergate-staging-eu-north-1-resources`` by default, but it can be changed using the configuration ``S3_BUCKET_NAME``.
```

Comment:
```suggestion
The bucket is named ``jobbergate-staging-eu-north-1-resources`` by default, but it can be changed using the configuration ``S3_BUCKET_NAME``.
```

Reply:
Thanks, suggestion addressed by [83d8247](https://github.com/omnivector-solutions/jobbergate/pull/114/commits/83d8247450779f3229bb9301acd8e7ce2afcaae5).